context/opencensus-context/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## 0.7.0
+Released 2019-07-31
+
```

Comment:
Was wondering if we should put "-version update" under the notes or leave it as blank?

Reply:
I don't have strong opinion, personally I would probably put "bumped version" here.

Reply:
Okay, seeing an empty space is awkward. I will make the change.